src/Factories/SegmentFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace CachetHQ\Segment\Factories;
+
+use Segment;
+use Segment_Client;
+
+class SegmentFactory
+{
+	/**
+	 * Make a new Segment client.
+	 *
+	 * @param array $config
+	 * @return Segment_Client
+	 */
+	public function make(array $config)
+	{
+		$config = $this->getConfig($config);
+
+		Segment::init(
```

Comment:
Yeah. Due to the Segement package setup, we'll have to drop the manager all together and just initialize the Segment init method.